src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderResourceIntTest.java

```diff
@@ -526,16 +532,12 @@ public void testCanAdjustReleasedPrisonerFromPrisonToHospital() {
         assertThat(releaseResponse.getStatusCodeValue()).isEqualTo(200);
 
         // check that no new movement is created
-        final var movementCheck1 =  testRestTemplate.exchange(
-            "/api/bookings/{bookingId}/movement/{sequenceNumber}",
-            GET,
-            createHttpEntity(token, null),
-            new ParameterizedTypeReference<String>() {
-            },
-            bookingId, 2
-        );
-
-        assertThatJsonFileAndStatus(movementCheck1, 200, "movement_discharge_1.json");
+        val latestMovement = movementsRepository.getMovementsByOffenders(List.of(offenderNo.toString()), null, true, false ).get(0);
```

Comment:
val? In Java? I need to read some release notes...

Reply:
lombok